pkg/service/service.go

```diff
@@ -230,11 +223,20 @@ func (s *Service) UpsertService(params *UpsertServiceParams) (bool, lb.ID, error
 
 		logfields.SessionAffinity:        params.SessionAffinity,
 		logfields.SessionAffinityTimeout: params.SessionAffinityTimeoutSec,
+
+		logfields.LoadBalancerSourceRanges: params.LoadBalancerSourceRanges,
 	})
 	scopedLog.Debug("Upserting service")
 
+	if !option.Config.EnableLoadBalancerSourceRangesCheck &&
+		len(params.LoadBalancerSourceRanges) != 0 {
+		scopedLog.Warnf("--%s is disabled, ignoring loadBalancerSourceRanges",
+			option.EnableLoadBalancerSourceRangesCheck)
```

Comment:
How frequently will this warning show up? If you have dozens of services with source-ranges, will we flood the logs for instance?

Reply:
This will be logged each time a service with source-ranges is updated. If it becomes an issue, we can always rate-limit such logs, or just log only once. However, for now I'd prefer to keep it as it's, because a) it's a security feature, so I want to make it obvious to a user that the check will be ignored, b) most of the kube-proxy-replacement users will run with the feature being enabled, as it's enabled by default.